### PR TITLE
🔧  Pinterest Schema updated

### PR DESCRIPTION
**Pull request by rfortmann-ewolff**
## What
`ad_id` and `account_id` are non-existent in the destination (BigQuery). The schema has a typo for `ad_id` and `account_id`.

## How
The schemas have been updated.

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@harshithmullapudi please review this PR.

File: airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_account_analytics.json [hunk 1]
```diff
@@ -271,4 +274,4 @@
       "type": ["null", "number"]
     }
   }
-}
+}
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_account_analytics.json:277]: can you add new line here

File: airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_analytics.json [hunk 2]
```diff
@@ -271,4 +274,4 @@
       "type": ["null", "number"]
     }
   }
-}
+}
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_analytics.json:277]: same ^ newline

File: airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_group_analytics.json [hunk 3]
```diff
@@ -271,4 +274,4 @@
       "type": ["null", "number"]
     }
   }
-}
+}
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_group_analytics.json:277]: same ^ newline

File: airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/campaign_analytics.json [hunk 4]
```diff
@@ -271,4 +274,4 @@
       "type": ["null", "number"]
     }
   }
-}
+}
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/campaign_analytics.json:277]: same ^ newline

**Review by harshithmullapudi — CHANGES_REQUESTED** [review]

**Comment by harshithmullapudi** [comment]
Hey @rfortmann-ewolff  https://docs.airbyte.com/connector-development#updating-an-existing-connector can you check this

**Comment by rfortmann-ewolff** [comment]
> Hey @rfortmann-ewolff https://docs.airbyte.com/connector-development#updating-an-existing-connector can you check this

all tests pass:
- no new tests required
- no documentation update required

**Comment by harshithmullapudi** [comment]
Hey, @rfortmann-ewolff can you bump the version as mentioned here https://docs.airbyte.com/connector-development#publishing-a-connector. Also can you share integration tests and unit tests screenshot

**Comment by harshithmullapudi** [comment]
Hey @rfortmann-ewolff I tried running the sourceAcceptance tests and looks like they are failing 
Screenshot 2022-01-28 at 6 31 31 PM

1. docker pull airbyte/source-acceptance-test
2. ./gradlew :airbyte-integrations:connectors:source-pinterest:sourceAcceptanceTest

**Comment by rfortmann-ewolff** [comment]
> Hey @rfortmann-ewolff I tried running the sourceAcceptance tests and looks like they are failing Screenshot 2022-01-28 at 6 31 31 PM
> 
> 1. docker pull airbyte/source-acceptance-test
> 2. ./gradlew :airbyte-integrations:connectors:source-pinterest:sourceAcceptanceTest

I cannot reproduce the failing test - it's failing because of this:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':airbyte-integrations:bases:airbyte-protocol:generateProtocolClassFiles'.
> Process 'command 'bin/generate-protocol-files.sh'' finished with non-zero exit value 1
```

So I guess, I'll just change the `AD_ACCOUNT_ID` type (and other similar changes in the schemas) to the origin.

**Comment by harshithmullapudi** [comment]
You can rebase it with latest master first then

**Comment by rfortmann-ewolff** [comment]
> You can rebase it with latest master first then

Even after the rebase, I still get the same error.

**Comment by rfortmann-ewolff** [comment]
I only rebased on my fork and not on the origin master - my bad.

**Comment by rfortmann-ewolff** [comment]
> You can rebase it with latest master first then
```
configured_catalog = ConfiguredAirbyteCatalog(streams=[ConfiguredAirbyteStream(stream=AirbyteStream(name='boards', json_schema={'$schema': ...l: 'incremental'>, cursor_field=None, destination_sync_mode=<DestinationSyncMode.append: 'append'>, primary_key=None)])
allowed_empty_streams = set()

    def _validate_empty_streams(self, records, configured_catalog, allowed_empty_streams):
        """
        Only certain streams allowed to be empty
        """
        counter = Counter(record.stream for record in records)

        all_streams = set(stream.stream.name for stream in configured_catalog.streams)
        streams_with_records = set(counter.keys())
        streams_without_records = all_streams - streams_with_records

        streams_without_records = streams_without_records - allowed_empty_streams
>       assert not streams_without_records, f"All streams should return some records, streams without records: {streams_without_records}"
E       AssertionError: All streams should return some records, streams without records: {'ad_group_analytics', 'board_section_pins', 'boards', 'board_pins', 'ad_account_analytics', 'campaign_analytics', 'board_sections'}
E       assert not {'ad_account_analytics', 'ad_group_analytics', 'board_pins', 'board_section_pins', 'board_sections', 'boards', ...}

/usr/local/lib/python3.7/site-packages/source_acceptance_test/tests/test_core.py:289: AssertionError
```
I don't understand the error that is meant here

**Comment by rfortmann-ewolff** [comment]
Even though the schema change seemed easy and should not have produced much errors, I had to close my PR as tests were impossible to reproduce.

**Comment by rfortmann-ewolff** [comment]
If anyone wants to try to solve that schema change, here is where the typos take place:
https://github.com/airbytehq/airbyte/blob/98b5aabd187a7338e01114c373a397d3aae3590e/airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_account_analytics.json#L9

https://github.com/airbytehq/airbyte/blob/98b5aabd187a7338e01114c373a397d3aae3590e/airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_analytics.json#L9

https://github.com/airbytehq/airbyte/blob/98b5aabd187a7338e01114c373a397d3aae3590e/airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_group_analytics.json#L9

https://github.com/airbytehq/airbyte/blob/98b5aabd187a7338e01114c373a397d3aae3590e/airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/campaign_analytics.json#L9

**Pull request closed without merging**